Cargo.toml

```diff
@@ -163,8 +163,11 @@ flate2 = "1.0.26"
 fs-err = "2.9.0"
 futures = "0.3.28"
 gateway-client = { path = "gateway-client" }
-gateway-messages = { git = "https://github.com/oxidecomputer/management-gateway-service", default-features = false, features = ["std"], rev = "2dcc63c645005a7184ba754b5c42c341a0777c66" }
-gateway-sp-comms = { git = "https://github.com/oxidecomputer/management-gateway-service", rev = "2dcc63c645005a7184ba754b5c42c341a0777c66" }
+#gateway-messages = { git = "https://github.com/oxidecomputer/management-gateway-service", default-features = false, features = ["std"], rev = "2dcc63c645005a7184ba754b5c42c341a0777c66" }
+#gateway-sp-comms = { git = "https://github.com/oxidecomputer/management-gateway-service", rev = "2dcc63c645005a7184ba754b5c42c341a0777c66" }
+# TODO merge MGS branch and point back to a rev on main
```

Comment:
Note: fix this after https://github.com/oxidecomputer/management-gateway-service/pull/87 is merged

Reply:
fixed in [d5edcdf](https://github.com/oxidecomputer/omicron/pull/2997/commits/d5edcdfea72a234767e6a54fbb80928c0119ebbc)